gulpfile.babel.js

```diff
@@ -247,6 +248,15 @@ gulp.task('json', function() {
 });
 
 
+gulp.task('keycheck', () => fs.access('./service-account-key.json').catch(
+  error => Promise.reject(error +
```

Comment:
This line should be indented 4 spaces since it's a line continuation, so should the next line.

BTW, if you want to avoid all this indentation and implicit vs. explicit return stuff, you can use an async function. Something like this should work:

```js
gulp.task('keycheck', async () => {
  try {
    await fs.access('./service-account-key.json');
  } catch (err) {
    throw new Error('service-account-key.json not found.\n' +
        'See https://ga-dev-tools.appspot.com/embed-api/' +
        'server-side-authorization/ for details on how to create it.\n' +
        err.stack);
  }
});
```

Reply:
I considered that, but in this case I actually found I liked the promise thing better. I'm a fan of async functions in cases where there's elaborate control flow like loops and conditionals, but for straightforward stuff I like the conciseness of the current design.